packages/headless/src/controllers/insight/sort/headless-insight-sort.test.ts

```diff
@@ -0,0 +1,63 @@
+import {
+  MockInsightEngine,
+  buildMockInsightEngine,
+} from '../../../test/mock-engine';
+import {Sort, SortProps, buildSort} from './headless-insight-sort';
+import {
+  buildDateSortCriterion,
+  SortOrder,
+} from '../../../features/sort-criteria/criteria';
+import {updateSortCriterion} from '../../../features/sort-criteria/sort-criteria-actions';
+import {executeSearch} from '../../../features/insight-search/insight-search-actions';
+import {updatePage} from '../../../features/pagination/pagination-actions';
+import {configuration, sortCriteria} from '../../../app/reducers';
+
+describe('InsightSort', () => {
+  let engine: MockInsightEngine;
+  let props: SortProps;
+  let sort: Sort;
+
+  function initSort() {
+    sort = buildSort(engine, props);
+  }
+
+  beforeEach(() => {
+    engine = buildMockInsightEngine();
+    props = {
+      initialState: {},
+    };
+
+    initSort();
+  });
+
+  it('it adds the correct reducers to engine', () => {
+    expect(engine.addReducers).toHaveBeenCalledWith({
+      configuration,
+      sortCriteria,
+    });
+  });
```

Comment:
This test is not really needed because it is tested already in the core controller.